wandb/sdk/launch/builder/build.py

```diff
@@ -499,3 +509,75 @@ def _create_docker_build_ctx(
 def join(split_command: List[str]) -> str:
     """Return a shell-escaped string from *split_command*."""
     return " ".join(shlex.quote(arg) for arg in split_command)
+
+
+def make_image_uri(
```

Comment:
nit: since we're building the image here, we can name this function `build_image_with_builder` or something

Reply:
Yeah I made this function for more easily isolating this step while testing, totally did not think about the name for more than about 3 seconds.